source/stylesheets/utilities/_viewport.scss

```diff
@@ -0,0 +1,68 @@
+// Viewport
+//
+// Utilities that apply a rule based on some media query.
+//
+// Styleguide 5.18
+
+// u-onlyOnHomepageHeroSmall
+//
+// Shows its content only under a specific resolution that works for the Homepage's hero short.
+//
+// markup:
+// <div class="u-onlyOnHomepageHeroSmall">I only show under a certain resolution</div>
+//
+// Styleguide 5.18.1
+
+.u-onlyOnHomepageHeroSmall {
+  @include media('>795px') {
+    display: none !important;
+  }
+}
+
+// u-onlyOnHomepageHeroLarge
+//
+// Shows its content only above a specific resolution that works for the Homepage's hero short.
+//
+// markup:
+// <div class="u-onlyOnHomepageHeroLarge">I only show above a certain resolution</div>
+//
+// Styleguide 5.18.2
+
+.u-onlyOnHomepageHeroLarge {
+  @include media('<=795px') {
+    display: none !important;
+  }
+}
+
+// u-onlyResponsibleResponsiveDesignPicture
+//
+// Shows its content only between resolutions specific to the Responsible Responsive Design picture
+//
+// markup:
+// <div class="u-onlyResponsibleResponsiveDesignPicture">I only show between certain resolutions</div>
+//
+// Styleguide 5.18.3
+
+.u-onlyResponsibleResponsiveDesignPicture {
+  @include media('<=tablet') {
+    display: none;
+  }
+  @include media('>1035px') {
+    display: none;
+  }
+}
+
+// u-onlyAboveResponsibleResponsiveDesignPicture
+//
+// Shows its content only above a resolution specific to the Responsible Responsive Design picture
+//
+// markup:
+// <div class="u-onlyAboveResponsibleResponsiveDesignPicture">I only show above a certain resolution</div>
+//
+// Styleguide 5.18.4
+
+.u-onlyAboveResponsibleResponsiveDesignPicture {
+  @include media('<=1035px') {
+    display: none;
+  }
+}
```

Comment:
I can't find a reference to any of these classes in the PR. Where they extracted from blue?